ONSdigital/eq-questionnaire-runner
# Update error behaviour and summaries

### What is the context of this PR?
Updated runner to use design system 16.1.0.

**Two breaking changes:**
**Summaries** - updated the component implementation. This is in readiness for the multiple section headings.

**Errors** - Added an id of `answer.id` to the error config in answer.html and updated the id's of each answer type to be unique to this. This is to allow a different behaviour when clicking error links and the way it focusses on inputs.

### How to review 
- Check that all existing summaries display correctly and don't have any missing content.
- Check validation errors for the following behaviour:
  - When an error link is clicked it jumps to the error panel (not input inside)
  - When error is anything other than checkbox or radio it should focus the field
  - When error is checkbox or radio it jumps to the panel and nothing is focussed
  - When JS is off it jumps to the panel and nothing is focussed
